common/Makefile.am

```diff
@@ -1,2 +1,3 @@
 noinst_LIBRARIES = libpaf-common.a
 libpaf_common_a_SOURCES = paf-hwcap.c paf-interp.c paf-common.h paf-hwcap.h
+AM_CFLAGS = -fpic
```

Comment:
I wonder if the real problem is caused by the usage of libpaf-common.a...
AFAIR, libpaf-common.a was created as a workaround when we migrated the project to Automake. We wanted a common directory and we didn't want to build these objects twice.
Would [libtool convenience libraries](http://www.gnu.org/software/automake/manual/html_node/Libtool-Convenience-Libraries.html#Libtool-Convenience-Libraries) help us here?